spec/factories/users.rb

```diff
@@ -0,0 +1,10 @@
+FactoryGirl.define do
+  factory :user do
+    name 'Dogbert'
+    sequence(:email) { |i| 'fubert#{i}@example.org' }
```

Comment:
Unused block argument - `i`. You can omit the argument if you don't care about it.